components/table/nz-table.component.less

```diff
@@ -0,0 +1,27 @@
+@import '../style/themes/index';
```

Comment:
let's put all the style into patch.less in component/table/style

Reply:
Done.